lib/beaker-pe/install/pe_utils.rb

```diff
@@ -542,8 +547,18 @@ def generic_install hosts, opts = {}
             #Windows allows frictionless installs starting with PE Davis, if frictionless we need to skip this step
             elsif (host['platform'] =~ /windows/ && !(host['roles'].include?('frictionless')) || install_via_msi?(host))
               opts = { :debug => host[:pe_debug] || opts[:pe_debug] }
-              msi_path = "#{host['working_dir']}\\#{host['dist']}.msi"
-              install_msi_on(host, msi_path, {}, opts)
+              #DLU : msi_path = "#{host['working_dir']}\\#{host['dist']}.msi"
+
+              path = host['pe_dir'] || opts[:pe_dir]
+              version = host['pe_ver'] || opts[:pe_ver_win]
+              filename = "#{host['dist']}"
+              extension = ".msi"
+
+              path = "#{path}"
+              path.chomp!('/')
+              link = "#{path}/#{filename}#{extension}"
```

Comment:
I feel like these changes warrant more spec testing than what is currently in this PR